stock_move_backdating/models/stock_pack_operation.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Alex Comba - Agile Business Group
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models, _
+from odoo.exceptions import UserError
+
+
+def check_date(date):
+    now = fields.Datetime.now()
+    if date > now:

```

Comment:
@tafaRU What happen if you void the date_backdating field? Do you have an error?